Functions_Collections_Homework4.py

```diff
@@ -0,0 +1,35 @@
+from string import ascii_lowercase  # we will use it to generate keys as letters
+from random import randint, choice  # we will use it to generate random integers for dictionary values and list length; choice will be used to select random letter for keys
+
+
+# create new dicts where key = key from random list and value = list of values for this key
+def values_to_list(my_list):
```

Comment:
I decided to change my approach completely as initial code logic was quite hard to decompose. At the end I used your approach of taking list of values for each key and selecting the max of them, as I believe that's the best approach here. I didn't look into your code at all, just decided to implement the same idea)